cunumeric/deferred.py

```diff
@@ -32,6 +32,7 @@
     UnaryRedCode,
 )
 from .linalg.cholesky import cholesky
+from .sorting import sorting
```

Comment:
a minor quibble: why don't we just name things `sort` everywhere?

Reply:
There is no particular reasoning behind this - I will change it.